apps/package.json

```diff
@@ -208,6 +208,7 @@
     "filesaver.js": "0.2.0",
     "jszip": "3.0.0",
     "query-string": "4.1.0",
-    "react-hot-loader": "^1.3.1"
+    "react-hot-loader": "^1.3.1",
+    "selectize": "^0.12.4"
```

Comment:
:| I'm not sure why we have anything in "dependencies" vs "dev-depdendencies" now... I suppose it's a non-issue, but I sort of wish we'd standardize.